crypto/rand/drbg_lib.c

```diff
@@ -378,7 +378,7 @@ int RAND_DRBG_instantiate(RAND_DRBG *drbg,
         entropylen = drbg->get_entropy(drbg, &entropy, min_entropy,
                                        min_entropylen, max_entropylen, 0);
     if (entropylen < min_entropylen
-        || entropylen > max_entropylen) {
+            || entropylen > max_entropylen) {
```

Comment:
Is this extra indent optional or mandatory? And was it added by the indent script or by you? It's not that I dislike it (on the contrary), but I recall being told some time ago that the `||` should align after the opening brace, just like in argument lists (e.g. line 379). Also the Emacs indentation style is configured like that.

Reply:
This is optional, so I think it's an ok change to make, even though we don't *have* to.

As for automatic indentation, I don't know of any that supports this extra indentation.  That's my primary reason for not making it mandatory (I wouldn't want to fight Emacs *every damn time*).
If someone finds an indent program that does support this kind of construct, we're all ears!